l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol

```diff
@@ -226,6 +227,15 @@ contract MailboxFacet is ZkSyncStateTransitionBase, IMailbox {
     function _requestL2TransactionSender(
         BridgehubL2TransactionRequest memory _request
     ) internal nonReentrant returns (bytes32 canonicalTxHash) {
+        // Check that the transaction is allowed by the filterer (if the filterer is set).
+        if (s.transactionFilterer != address(0)) {
+            require(ITransactionFilterer(s.transactionFilterer).isTransactionAllowed(
```

Comment:
do you think it may make sense to require the entire `BridgehubL2TransactionRequest`? I am not sure about the impact on the cost of the transaction, but intuitively we might want to prevent users asking us for any additional parameters in the future for the tx validation

Reply:
For instance, access to `calldata`  may be the only option to prevent malicious contracts from bridging funds (as such txs would have `L1SharedBridge` as the sender)